gluten-ut/spark35/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -126,7 +126,7 @@ class VeloxTestSettings extends BackendTestSettings {
   enableSuite[GlutenHashExpressionsSuite]
   enableSuite[GlutenIntervalExpressionsSuite]
   enableSuite[GlutenJsonFunctionsSuite]
-    // Disable for Spark3.5.
+    // * in get_json_object expression not supported in velox
```

Comment:
Is there any issue to track this, if not please open one, thanks.

Reply:
Sure, I will open one, this is the PR where get_json_object support was added https://github.com/facebookincubator/velox/commit/20e467869b86d3c6d355cf52a550b94a10710479